nixos/modules/services/networking/expressvpn.nix

```diff
@@ -0,0 +1,20 @@
+{ config, lib, pkgs, ... }:
+
+{
+  options.services.expressvpn.enable = lib.mkOption {
+    type = lib.types.bool;
+    default = false;
+    description = ''
+      Enable the ExpressVPN daemon.
+    '';
+  };
+
+  config = lib.mkIf config.services.expressvpn.enable {
+    boot.kernelModules = [ "tun" ];
+
+    environment.systemPackages = [ pkgs.expressvpn ];
+    systemd.packages = [ pkgs.expressvpn ];
```

Comment:
That option is wrong. We don't want to add it everywhere.

Reply:
Didn't understand. What should I use instead? I was following https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/networking/mozillavpn.nix

Reply:
That gets added to every systemd unit which we don't want. https://search.nixos.org/options?channel=unstable&show=systemd.packages&from=0&size=50&sort=relevance&type=packages&query=systemd.packages+

Reply:
Only add it to the specific systemd unit that needs it.

Reply:
@SuperSandro2000 implemented the change, can you please review now.